docs/install-olm.md

```diff
@@ -0,0 +1,64 @@
+# How do I install OLM?
+
+The OLM is installable on Kubernetes clusters. It is already installed and available by default on 4.x OpenShift 
+clusters. For the following instructions to work, you must have a Kubernetes cluster running and the `kubectl` 
```

Comment:
Is there a reason each of these lines are a newline?

Reply:
You should bring this up with the rest of the team so we can decide if this is the approach we wish to take with all documents in `olm-book`.